SignUp/test_signup.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from SignUp import Register
+
+class TestSignUp(unittest.TestCase):
+    def setUp(self):
+        self.register = Register()
+        self.a = len(self.register.details)
```

Comment:
Am not seeing `self.e` being used anywhere in the code and please provide descriptive variable names